Frontend/src/components/UserProtectWrapper.tsx

```
@@ -0,0 +1,75 @@
+import axios from "axios";
+
+import { ReactNode, useContext, useEffect, useState } from "react";
+import { useNavigate } from "react-router";
+
+type Props = {
+    children: ReactNode;
+}
+
+const UserProtectWrapper = ({ children }: Props) => {
+    const token = localStorage.getItem("auth-token");
+    const [isLoading, setIsLoading] = useState<Boolean>(true);
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)